Gifski/util.swift

```diff
@@ -2383,3 +2383,21 @@ extension NSControl {
 		window?.makeFirstResponder(self)
 	}
 }
+
+extension URL {
+	enum MetadataKey {
+		case itemCreator
+
+		var attributeKey: String {
+			switch self {
+			case .itemCreator:
+				return kMDItemCreator as String
+			}
+		}
+	}
+
+	func setMetadata<T>(key: MetadataKey, value: T) throws {
+		let attributes = ExtendedAttributes(url: self)
```

Comment:
This is not needed: https://github.com/sindresorhus/Gifski/pull/93/files/ccf4101d69900875455a2cc47b2a4f3f5d87de1b..b1edcb12d7bae8acaefb105a49dffe2572d04995#diff-bf8c6e8ce59f7cf7b4011ef15b26bf7dR166